services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -278,13 +321,24 @@ async def stop_computation_project(
         if is_pipeline_running(pipeline_state):
             await scheduler.stop_pipeline(comp_task_stop.user_id, project_id)
 
+        # get run details if any
+        last_run: Optional[CompRunsAtDB] = None
+        with contextlib.suppress(ComputationalRunNotFoundError):
+            last_run = await computation_runs.get(
+                user_id=comp_task_stop.user_id, project_id=project_id
+            )
+
         return ComputationTaskGet(
             id=project_id,
             state=pipeline_state,
             pipeline_details=await compute_pipeline_details(
                 complete_dag, pipeline_dag, tasks
             ),
-            url=f"{str(request.url).rstrip(':stop')}",
+            url=parse_obj_as(AnyHttpUrl, f"{str(request.url).rstrip(':stop')}"),
```

Comment:
Careful [``rstrip``](https://docs.python.org/3/library/stdtypes.html#str.rstrip) remove trailing space or the characters list in ``:stop`` (not necessarily the word!). In 3.9 there is ``str.removesuffix`` but for now I would use instead
```python
f"{request.url}".replace(":stop", "")
```